aggregator/src/aggregator.rs

```diff
@@ -2293,6 +2382,49 @@ impl VdafOps {
             .await?)
     }
 
+    /// Handle requests to the helper to delete an aggregation job.
+    async fn handle_aggregate_delete_generic<
+        const SEED_SIZE: usize,
+        Q: AccumulableQueryType,
+        A: vdaf::Aggregator<SEED_SIZE, 16>,
+        C: Clock,
+    >(
+        datastore: &Datastore<C>,
+        task: Arc<AggregatorTask>,
+        aggregation_job_id: &AggregationJobId,
+    ) -> Result<(), Error>
+    where
+        A: 'static + Send + Sync,
+        A::AggregationParam: Send + Sync,
+        A::AggregateShare: Send + Sync,
+        for<'a> A::PrepareState: Send + Sync + Encode + ParameterizedDecode<(&'a A, usize)>,
+        A::PrepareShare: Send + Sync,
+        A::PrepareMessage: Send + Sync,
+        A::OutputShare: Send + Sync,
+    {
+        Ok(datastore
+            .run_tx("delete aggregation job", |tx| {
```

Comment:
```suggestion
            .run_tx("delete_aggregation_job", |tx| {
```
nit: consistent with our other tx names.